### PR TITLE
Contact me

### PR DESCRIPTION
**Pull request by abiy006**
Tasks accomplished by this PR

```
- Contact me layout is added
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.
```

### CODE REVIEW

**Review by BregornOriginal — COMMENTED** [review]
## Hi @abiy006  

## While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have some commit history on your Pull Request about working in pair programming. Kindly request your partner to make some changes to the project, so we are able to review it 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @abiy006 @BregornOriginal,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 1]
```diff
@@ -325,6 +311,20 @@ main button {
...
 }
 
+section.contact_me button {
+  width: 142px;
+  height: 48px;
+  color: #6070ff;
+  background: #fff;
+  border-radius: 8px;
+  border: 1px solid #fff;
+  font-family: 'Poppins', sans-serif, Courier, monospace;
```
Comment (cancelei) [inline style.css:321]: Here and in other parts, try changing PX to Rem, so this part becomes more responsive.

Each 1 rem = 16px when converting.

Avoid using defined height/width in px, try using 100% or auto, in the last case 100vh (for height) and 100wh(for width).

Comment (abiy006) [reply style.css:321]: @cancelei - thanks for your feedback, I will optimize based on the comments

**Pull request merged**